hack/k3d.mk

```diff
@@ -1,17 +1,17 @@
 CLUSTER_NAME ?= kyma
 REGISTRY_PORT ?= 5001
-REGISTRY_NAME ?= ${CLUSTER_NAME}-registry
+REGISTRY_NAME ?= ${CLUSTER_NAME}-registry.localhost
 
 ifndef PROJECT_ROOT
 $(error PROJECT_ROOT is undefined)
 endif
-include $(PROJECT_ROOT)/hack/tools.mk
 
 ##@ K3D
 
 .PHONY: create-k3d
-create-k3d: kyma ## Create k3d with kyma CRDs.
-	${KYMA} provision k3d --registry-port ${REGISTRY_PORT} --name ${CLUSTER_NAME} --ci -p 6080:8080@loadbalancer -p 6433:8433@loadbalancer
+create-k3d: delete-k3d ## Delete old k3d registry and cluster. Create preconfigured k3d with registry
```

Comment:
The kyma cli used to delete old k3d and registry, As I used plain `k3d` cli I added `delete-k3d` as dependency to `create-k3d ` target